.github/actions/e2e-quantic/action.yml

```diff
@@ -0,0 +1,33 @@
+name: "Quantic E2E tests"
+description: "Runs Cypress tests for Quantic"
+runs:
+  using: composite
+  steps:
+    - run: |
+        ./node_modules/.bin/sfdx force:auth:jwt:grant \
+          --clientid ${{ secrets.SFDX_AUTH_CLIENT_ID }} \
+          --jwtkeyfile ${{ secrets.SFDX_AUTH_JWT_KEY }} \
+          --username rdaccess@coveo.com \
+          --instanceurl https://login.salesforce.com \
+          --setdefaultdevhubusername
+      shell: bash
+      with:
+        working-directory: ./packages/quantic
```

Comment:
Not sure why syntax highlighting looks wrong for those three lines. In VSCode, they have the correct syntax highlighting.